static/app/views/issueDetails/streamline/eventTitle.tsx

```diff
@@ -1,4 +1,15 @@
-import {type CSSProperties, forwardRef, Fragment} from 'react';
+import 'intersection-observer'; // polyfill
```

Comment:
Do we want to include the polyfill? It's [pretty well supported](https://caniuse.com/intersectionobserver) -- I feel like we use CSS that is less supported than that 😅

Reply:
@billyvg I can remove the polyfill. I was copying how we use `IntersectionObserver` in other places.

Reply:
Ah let's leave it then if we're using polyfill elsewhere